istioctl/pkg/rbac/testdata/rbac-policies-v2.golden.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: rbac.istio.io/v1alpha1
+kind: ServiceRole
+metadata:
+  name: service-viewer
+  namespace: default
+spec:
+  rules:
+  - constraints:
+    - key: destination.labels[version]
```

Comment:
Our docs at https://preliminary.istio.io/docs/reference/config/authorization/istio.rbac.v1alpha1/#AccessRule say that `services` is required.  This file, which I assume is the output of the test, lacks services.  The input had `["productpage.svc.cluster.local"]`.  Is the documentation correct?

Reply:
Yes it's correct. RBAC 2 will use workload selector instead. So `services` will be deprecated.